src/components/Navbar/index.js

```diff
@@ -31,7 +31,7 @@ export default function Navbar() {
           </Link>
         </li>
         <li>
-          <Link href="/#about">Sobre</Link>
+          <Link href="/AboutUs">Sobre</Link>
```

Comment:
Apesar de ser "Sobre nós", acredito que fique mais adequado no link ser apenas "about", além de menor para a url, também evita de usar caracteres maiúsculos. Lembrar que após atualizar a branch o componente Navbar vai sumir (virou Header).
```suggestion
          <Link href="/about">Sobre nós</Link>
```